src/components/Schedule.js

```diff
@@ -34,68 +34,79 @@ const Schedule = () => {
+                                <div className="schedule__category">
+                                  <p className="schedule__category__time">
+                                    {event.time}
+                                  </p>
                                   <p
-                                    className="category__type"
+                                    className="schedule__category__type"
                                     style={{ color: event.theme }}
                                   >
                                     {event.type}
                                   </p>
                                 </div>
-                              </td>
-                              {/* TODO: Need to change hardcoded width here */}
-                              <td width="450px">
-                                <div className="location">
-                                  <p className="location__title">
+                                <div className="schedule__location">
+                                  <p
+                                    className="schedule__location__title"
+                                    title={event.title}
+                                  >
                                     {event.title}
                                   </p>
                                   <a
-                                    className="location__zoom"
+                                    className="schedule__location__zoom"
                                     href={event.location}
+                                    title={event.location}
+                                    // make the link open in a new tab
+                                    target="_blank"
+                                    rel="noopener noreferrer"
                                   >
                                     {event.location}
                                   </a>
                                 </div>
                               </td>
-                              {/* TODO: Need to change hardcoded width here */}
-                              <td width="300px">
-                                <div className="audience">
-                                  <p
-                                    className={
-                                      event.audience ? "audience__type" : ""
-                                    }
-                                  >
-                                    {event.audience}
-                                  </p>
-                                  <p
-                                    className={
-                                      event.company ? "audience__company" : ""
-                                    }
-                                  >
-                                    {event.company}
-                                  </p>
-                                </div>
+                              <td className="schedule__audience">
```

Comment:
do we want to hide the beginner tag? Im actually not sure since we don't have the mobile design

Reply:
I figured we could put everything else in the expandable row